.github/workflows/pr.yml

```diff
@@ -30,12 +30,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       name: Checkout che-theia source code
+      with:
+        fetch-depth: "0"
     - uses: actions/setup-node@v1
       name: Configuring nodejs 10.x version
       with:
         node-version: '12.x'
     - name: build
       run: yarn
+    - name: Test Coverage
+      uses: codecov/codecov-action@v1
```

Comment:
no need to specify location of coverage reports, it's finding it automatically ?

Reply:
From what I saw from logs, coverage reports are found automatically:

```
==> Searching for coverage reports in:
    + .
    -> Found 12 reports
==> Detecting git/mercurial file structure
==> Reading reports
    + ./generator/coverage/clover.xml bytes=40754
    + ./generator/coverage/coverage-final.json bytes=111291
    + ./generator/coverage/lcov.info bytes=15992
    + ./plugins/workspace-plugin/coverage/clover.xml bytes=32467
    + ./plugins/workspace-plugin/coverage/coverage-final.json bytes=102618
    + ./plugins/workspace-plugin/coverage/lcov.info bytes=14644
    + ./plugins/resource-monitor-plugin/coverage/clover.xml bytes=9776
    + ./plugins/resource-monitor-plugin/coverage/coverage-final.json bytes=27324
    + ./plugins/resource-monitor-plugin/coverage/lcov.info bytes=3888
    + ./plugins/ports-plugin/coverage/clover.xml bytes=26398
    + ./plugins/ports-plugin/coverage/coverage-final.json bytes=84172
    + ./plugins/ports-plugin/coverage/lcov.info bytes=12489
==> Appending adjustments
    https://docs.codecov.io/docs/fixing-reports
    + Found adjustments
==> Gzipping contents
        88K	/tmp/codecov.kOCN4l.gz
```